### PR TITLE
mounter: don't build delete dml when set index key

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
don't build delete dml when set index key

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/entry/mounter.go [hunk 1]
```diff
@@ -119,6 +119,10 @@ func (m *Mounter) mountRowKVEntry(row *rowKVEntry) (*model.DML, error) {
 }
 
 func (m *Mounter) mountIndexKVEntry(idx *indexKVEntry) (*model.DML, error) {
+	// skip set index KV
+	if !idx.Delete {
+		return nil, nil
```
Comment (suzaku) [inline cdc/entry/mounter.go]: Please add a test case to help reviewers understand what this change tried to fix.

Comment (leoppro) [reply cdc/entry/mounter.go]: this change fixes nothing. We translate setting row key to `replace` query, so we needn't build delete dml for setting index key.

Comment (leoppro) [reply cdc/entry/mounter.go]: see:https://docs.google.com/document/d/1D1IqIs_q0y0YSnu6hJgKZO4lkCkDqOexDxzMdK-7NRU/edit#

**Pull request merged**